example-apps/kitchen-sink-new-router/source/components/stress-tester/stress-tester.ts

```diff
@@ -13,39 +13,70 @@ import {FORM_DIRECTIVES, NgForm, NgIf} from '@angular/common';
 class StressItem {
 }
 
-//
+@Component({
+  selector: 'stress-rec-item',
+  inputs: ['value'],
+  directives: [NgIf, StressRecItem],
```

Comment:
@ericjim do we still need to include NgIf here? I might be out of date, but I thought that those are available by default.